bridge/qb/shared/main.lua

```diff
@@ -1,15 +1,17 @@
-local starterItems = require 'config.server'.starterItems
 local qbShared = require 'shared.main'
 
----@deprecated use starterItems in config/server.lua
-qbShared.StarterItems = {}
+if lib.context == 'server' then
```

Comment:
```suggestion
if IsDuplicityVersion() then
```
I believe this is functionally the same. I've not seen lib.context used and it's not specifically mentioned in the ox_lib docs so it may be intended for internal use only.